bats/tests/containers/factory-reset.bats

```diff
@@ -156,22 +156,22 @@ rdctl_factory_reset() {
 
 check_directories() {
     # Check if all expected directories are created after starting application/ are deleted after a factory reset
-    delete_dir=("$PATH_CONFIG")
+    delete_dir=("$PATH_CONFIG_FILE" "$PATH_LOGS" "$PATH_APP_HOME/credential-server.json" "$PATH_APP_HOME/rd-engine.json")
```

Comment:
Let's move `PATH_CONFIG_FILE` into the `is_windows` branch:

It is redundant for unix because we already delete `PATH_CONFIG` and it helps to clarify the comment about `PATH_CONFIG` being the same as `PATH_APP_HOME` (so we only delete the file, and not the directory).